lib/modules/manager/crossplane/readme.md

```diff
@@ -0,0 +1,40 @@
+The `crossplane` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
+This is because there is no commonly accepted file/directory naming convention for crossplane YAML files and we don't want to check every single `*.yaml` file in repositories just in case any of them have Crossplane Packages definitions: Configurations, Providers, Functions.
+
+If most `.yaml` files in your repository are Crossplane ones, then you could add this to your config:
+
+```json
```

Comment:
```
json title="fileMatch if most .yaml files are for Crossplane"
```

First read my _other_ comment about this config example. Then fix up the title if needed.